### PR TITLE
www.orefolder.net

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/109445

<details><summary>Screenshot:</summary>

Снимок экрана 2022-02-22 в 19 18 30

Снимок экрана 2022-02-22 в 19 19 08

Снимок экрана 2022-02-22 в 19 19 01

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/subscriptions.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 michalpasterski.pl##.newsletter-popup
 thestreet.com##.bcpContentTemplate
 fotospor.com##.alert-message
+orefolder.net##.followme
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/subscriptions.txt:41]: They are links to their social media. @zloyden

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -39,6 +39,7 @@ w.atwiki.jp###wikibody > h3[id^="id_"] + br + hr
 ieeebd.org##.b-r--before-site-content
 ieeebd.org##div[style="min-height: 280px;"]
 logmi.jp##.ranking-item[data-rank="PR"]
+orefolder.net##.sidebar > .widget_text
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:42]: This is self-promo.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 3]
```diff
@@ -34,6 +34,7 @@ academia.edu#$#.modal-backdrop { display: none!important; }
 academia.edu#$#body { overflow: auto!important; }
 hatalike.jp##.hdr_app_wrap
 clubdam.com##.bnr-app
+orefolder.net##.appdl
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/mobile-app.txt:37]: Hmm, I don't see this element. Is this in the issue URL?

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `orefolder.net##div[style$="min-height:175px;"]` to JPF.

**Review by Yuki2718 — COMMENTED** [review]
Also `orefolder.net#?#.custom-html-widget:has(> a[href^="https://www.amazon.co.jp/dp/"])`

File: AnnoyancesFilter/sections/subscriptions.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/subscriptions.txt]: @Yuki2718 can you please translate this?
```
の最新情報をお届けします
```
As I understand, it's just their social links without any recommendations like `Find us here...`.
@jellizaveta we can move this rule to the `annoyances.txt`.

Comment (Yuki2718) [reply AnnoyancesFilter/sections/subscriptions.txt]: Maybe "These bring you up-to-date information about OREFOLDER."

Comment (zloyden) [reply AnnoyancesFilter/sections/subscriptions.txt]: Hm, it is for `annoyances.txt`.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**